### ISSUE TITLE
continuous issue about rendering the sample

### ISSUE THREAD

**Issue by sygyq305** [GuyTevet/MotionCLIP#26]
![image](https://github.com/GuyTevet/MotionCLIP/assets/52557089/d568e958-de4b-472f-a7c6-c49f61a67f6b)
In fig5, each motion has many actions of fragment. Example of fragment is as follows.
![image](https://github.com/GuyTevet/MotionCLIP/assets/52557089/0ddbb34a-f0fa-4344-a1fe-68a8fe2b6b48)
How are the actions of this fragment extracted from the results?

**Comment by sygyq305** [comment]
The final output file is gif. Can I change the format to mp4.

**Comment by GuyTevet** [comment]
Yes, [here](https://github.com/GuyTevet/MotionCLIP/blob/0b32d39ef3b237c0f0dd02cd779c83829dc80527/src/visualize/anim.py#L148)

**Comment by sygyq305** [comment]
I want to change "gen_{}_{}.gif" to mp4.
Apart from changing the format of save_path, what else needs to be modified？
